src/garage/torch/distributions/base.py

```diff
@@ -0,0 +1,37 @@
+class Distribution:
```

Comment:
We don't need this class for pytorch. We can just use pytorch distributions. https://pytorch.org/docs/stable/distributions.html

Reply:
Thank you for letting me know.
So This means that I don't need to return "dist" from GaussianMLPModule?

Reply:
I'll remove the Distribution class and use Pytorch distributions.

Reply:
Oh sorry for the confusion. I just meant that we don't need to write our own `Distribution` class in garage and we should be able to use Pytorch's directly and only implement those in-exist in PyTorch.